rosserial_server/include/rosserial_server/serial_session.h

```diff
@@ -45,81 +45,63 @@
 namespace rosserial_server
 {
 
-class SerialSession : public Session<boost::asio::serial_port>
+class SerialSession
 {
 public:
   SerialSession(boost::asio::io_service& io_service, std::string port, int baud)
```

Comment:
Realistically, this is no longer a "Session", it's now a session manager. However, for the sake of not breaking API for other nodes which embed rosserial_server, it will remain as it is.